src/bokeh/sphinxext/_templates/bokehjs_codepen_init.html

```diff
@@ -10,65 +10,63 @@
 {% for js_file in js_files %}
 <script type="text/javascript" src="{{js_file}}"></script>{% endfor %}
 
+
+
 <script>
-    (function () {
+  (function () {
 
-        // this monkey patches Plotting.show to insert the plot below
-        // the script block
+    // this monkey patches Plotting.show to insert the plot below
+    // the script block
```

Comment:
@mattpap this code for this sphinx extension is very old. Is this monkey patch still needed? What would it take to be able to avoid it (sometime in another PR in the future)

Reply:
It looks like it's unnecessary, because `show()` does something similar, but I think equivalent, when `target` is not provided. It would be worthwhile to test this without this hack.